Modules/ITS/src/ITSClusterTask.cxx

```diff
@@ -153,34 +155,44 @@ void ITSClusterTask::monitorData(o2::framework::ProcessingContext& ctx)
     const auto& ROF = clusRofArr[iROF];
     const auto bcdata = ROF.getBCData();
     int nClustersForBunchCrossing = 0;
+    int nLongClusters[ChipBoundary[NLayerIB]] = { 0 };
+    int nHitsFromClusters[ChipBoundary[NLayerIB]] = { 0 }; // only IB is implemented at the moment
+
     for (int icl = ROF.getFirstEntry(); icl < ROF.getFirstEntry() + ROF.getNEntries(); icl++) {
 
       auto& cluster = clusArr[icl];
       auto ChipID = cluster.getSensorID();
       int ClusterID = cluster.getPatternID(); // used for normal (frequent) cluster shapes
       int lay, sta, ssta, mod, chip, lane;
 
-      if (ChipID != ChipIDprev) {
+      if (ChipID != -1) { // TODO: is this needed?
```

Comment:
Idea of this thing was to avoid re-calculation of the mod/lane/ChipId each new cluster. If I am not wrong,  request to mGeom could take quite time